Update Laravel React CRUD post to use CLI for setup

mraible opened the pull request:
Updates [Build a Basic CRUD App with Laravel and React](https://developer.okta.com/blog/2018/12/06/crud-app-laravel-react) to streamline setup.

mraible commented:
I think this one has two options:

1. Leave it using @okta/okta-react v1 and add a note that you have to log in to the Admin Console and enable the implicit flow.
2. Upgrade it to the latest Okta React and update the code so it works with that version.

aaronpk commented:
This post also has the same problem as #679 where the Laravel version installed now will be too new for the instructions that assume an older version.

mraible commented:
@aaronpk I updated the code and this post to use Okta React 4.1.0. I did this because I believe the previous version used did not support PKCE, which means the app that CLI creates wouldn't work. I did NOT QA, I just updated the code.

aaronpk commented:
Same problem as the other one, it looks like the libraries used in the post are too old to work with PHP 7.4. They should work with an older PHP, but ultimately we'll need to update this post if we want it to work with 7.4 in the future.
